pom.xml

```diff
@@ -276,6 +276,19 @@
 
 	<build>
 		<plugins>
+			<plugin>
```

Comment:
NB: this needs to be before the spring-boot-maven-plugin as both execute in the same phase.
See this [link](https://docs.spring.io/spring-boot/docs/current/maven-plugin/reference/htmlsingle/#packaging.examples.custom-classifier) for details.

Reply:
I like the approach. I have added a comment on this (as this could be a source of ever-lasting questions otherwise ;) ) and pushed it to your branch.